web-services/security/pom.xml

```diff
@@ -82,7 +82,7 @@
         </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
-            <artifactId>bcprov-jdk16</artifactId>
+            <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
Was this a downgrade? Why was this change made?

Reply:
Reverted this change.